internal/resource/config/serversettings/server_settings_system_keys_resource.go

```diff
@@ -0,0 +1,396 @@
+						Required:    false,
+					},
+					"encrypted_key_data": schema.StringAttribute{
+						Description: "The system key encrypted.",
+						Computed:    true,
+						Optional:    true,
+						PlanModifiers: []planmodifier.String{
+							stringplanmodifier.UseStateForUnknown(),
+						},
+					},
+					"key_data": schema.StringAttribute{
+						Description: "The clear text system key base 64 encoded. The system key must be 32 bytes before base 64 encoding",
+						Computed:    true,
+						Optional:    false,
+						PlanModifiers: []planmodifier.String{
+							stringplanmodifier.UseStateForUnknown(),
+						},
+					},
+				},
+			},
+			"previous": schema.SingleNestedAttribute{
+				Description: "Previous SystemKeys Secrets that are used in cryptographic operations to generate and consume internal tokens.",
+				Computed:    true,
+				Optional:    true,
+				PlanModifiers: []planmodifier.Object{
+					objectplanmodifier.UseStateForUnknown(),
+				},
+				Attributes: map[string]schema.Attribute{
+					"creation_date": schema.StringAttribute{
+						Description: "Creation time of the key.",
+						Computed:    true,
+						Optional:    false,
+						Required:    false,
+					},
+					"encrypted_key_data": schema.StringAttribute{
+						Description: "The system key encrypted.",
+						Computed:    true,
+						Optional:    true,
+						PlanModifiers: []planmodifier.String{
+							stringplanmodifier.UseStateForUnknown(),
+						},
+					},
+					"key_data": schema.StringAttribute{
+						Description: "The clear text system key base 64 encoded. The system key must be 32 bytes before base 64 encoding",
+						Computed:    true,
+						Optional:    false,
+						PlanModifiers: []planmodifier.String{
+							stringplanmodifier.UseStateForUnknown(),
+						},
+					},
+				},
+			},
+			"pending": schema.SingleNestedAttribute{
+				Description: "Pending SystemKeys Secrets that are used in cryptographic operations to generate and consume internal tokens.",
+				Required:    true,
+				PlanModifiers: []planmodifier.Object{
+					objectplanmodifier.UseStateForUnknown(),
+				},
+				Attributes: map[string]schema.Attribute{
+					"creation_date": schema.StringAttribute{
+						Description: "Creation time of the key.",
+						Computed:    true,
+						Optional:    false,
+						Required:    false,
+					},
+					"encrypted_key_data": schema.StringAttribute{
+						Description: "The system key encrypted.",
+						Computed:    true,
+						Optional:    true,
+						PlanModifiers: []planmodifier.String{
+							stringplanmodifier.UseStateForUnknown(),
+						},
+					},
+					"key_data": schema.StringAttribute{
+						Description: "The clear text system key base 64 encoded. The system key must be 32 bytes before base 64 encoding",
+						Computed:    true,
+						Optional:    false,
+						PlanModifiers: []planmodifier.String{
+							stringplanmodifier.UseStateForUnknown(),
+						},
+					},
+				},
+			},
+		},
+	}
+
+	resp.Schema = schema
```

Comment:
If you don't need to make any changes to schema down here you can just make this the top line of the method. `resp.Schema = schema.Schema{...`. Rather than saving it to a separate variable like this.